tests/java/org/pantsbuild/tools/ivy/DependencyUpdateCheckerTest.java

```diff
@@ -0,0 +1,160 @@
+// Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package org.pantsbuild.tools.ivy;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.fail;
+
+public class DependencyUpdateCheckerTest {
+  private static final String SETTINGS_FILE_FLAG = "-settings";
+  private static final String IVY_FILE_FLAG = "-ivy";
+  private static final String SHOW_TRANSITIVE_FLAG = "-show-transitive";
+  private static final String CONFS_FLAG = "-confs";
+
+  private final List<String> testArgs = new ArrayList<>();
+
+  @Before
+  public void setUp() throws Exception {
+    DependencyUpdateChecker.setCallSystemExitOnFinish(false);
+
+    testArgs.clear();
+    testArgs.add(SETTINGS_FILE_FLAG);
+    testArgs.add("tests/resources/org/pantsbuild/tools/ivy/ivysettings.xml");
+    // TODO: Do we want to set "ivy.cache.dir" for these tests?
```

Comment:
I think it'd make sense. That way we wouldn't be reusing the same cache for the tests. How hard would that be?